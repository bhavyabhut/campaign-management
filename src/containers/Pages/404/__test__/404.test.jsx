import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter, MemoryRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';
import PageNotFound from '../index';
import renderer from 'react-test-renderer';

it('No Data Text Testing', () => {
  const { container } = render(<PageNotFound />, { wrapper: MemoryRouter });
  expect(container.querySelector('a')).toHaveTextContent('dashboard');
});

it('No Page Found match snapshot', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <PageNotFound />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
