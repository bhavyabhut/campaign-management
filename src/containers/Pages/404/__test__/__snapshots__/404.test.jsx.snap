// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Page Found match snapshot 1`] = `
<div
  className="noPage"
>
  <svg
    className="noPageLogo"
  >
    undraw_page_not_found_su7k.svg
  </svg>
  <span>
    Go back to 
    <a
      href="/dashboard"
      onClick={[Function]}
    >
      dashboard
    </a>
     
  </span>
</div>
`;
